api/src/main/scala/com/tersesystems/echopraxia/plusscala/api/FieldBuilder.scala

```diff
@@ -9,213 +9,6 @@ trait HasName {
   type Name
 }
 
-trait ValueTypeClasses {
```

Comment:
Move ValueTypeClasses and other "not fieldbuilder" classes from FieldBuilder.scala